app/models/order.rb

```diff
@@ -43,11 +43,27 @@ def change_status(new_status)
   def subtract_price
     return if dishes_count == 0
     dishes.each do |dish|
-      dish.subtract shipping / dishes_count, user
+      dish.subtract shipping / sum_users, user
     end
   end
 
   def from_today?
     date.today?
   end
+
+  def sum_users
+    dishes
+      .joins(:user_dishes)
+      .pluck('user_dishes.user_id')
+      .uniq
+      .count
+  end
```

Comment:
A method should be named after what it returns, not what it does. So this should be named something like `ordering_users_count`